python/lsst/pipe/tasks/photoCal.py

```diff
@@ -52,13 +52,11 @@ class PhotoCalConfig(pexConf.Config):
     )
     applyColorTerms = pexConf.Field(
         dtype=bool,
-        default=None,
-        doc=("Apply photometric color terms to reference stars? One of:\n"
-             "None: apply if colorterms and photoCatName are not None;\n"
-             "      fail if color term data is not available for the specified ref catalog and filter.\n"
-             "True: always apply colorterms; fail if color term data is not available for the\n"
-             "      specified reference catalog and filter.\n"
-             "False: do not apply."),
+        default=False,
+        doc=("Apply photometric color terms to reference stars?\n"
+             "True: always apply colorterms; fail if color term data is not available for the "
+             "specified reference catalog and filter.\n"
```

Comment:
This doc string line would be 120 characters long. Would it look nicer to maintain the indented look (indenting to match `True: ` above) as before?

Reply:
This spawned a whole conversation in #dm-docs-support. The challenge is balancing the `config.save` output vs. the rendered HTML that goes in pipelines.lsst.io. I think I've found a good balance of those here.
https://lsstc.slack.com/archives/C2B6DQBAL/p1667327052273339

Reply:
Had you previously removed the `always` in the `True` section here? If so, it looks like it has reappeared. 

I still think it would read better if written as `attempt to apply color terms; fail if...`. If instead you simply write `apply color terms; fail if...`, that implies that you a) are applying color terms, but then b) are failing to apply color terms, because none are available. Logically, this doesn't make sense to me, hence the need for an admission of an `attempt` being made.

Follow up point: use of `colorterms` vs `color terms`. I think the former should be reserved for reference to the config parameter, whilst the latter reserved for discussion of "color terms". I think this sentence probably should make use of `color terms` (with a space), but happy to defer to you if you were instead meaning to refer to the config parameter.

Reply:
Ok, I think I see what you mean. I've taken that wording.